tests/test_distributions.py

```diff
@@ -15,6 +15,15 @@ def random_variable_args():
         ("Bernoulli", {"probs": 0.5, "sample": 1.0}),
         ("Beta", {"concentration0": 1, "concentration1": 1}),
         ("Binomial", {"total_count": 5.0, "probs": 0.5, "sample": 1.0}),
+        (
+            "BetaBinomial",
+            {
+                "total_count": 10.0,
+                "concentration0": 1.0,
+                "concentration1": 2.0,
+                "sample": [3.0, 7.0],
```

Comment:
Great! If you want to make sure that things are working properly, you could add a value to the sample list that is outside of the bounds, the distribution `log_prob` should return Nan or -inf in those cases

Reply:
I tried it out but ``log_prob`` returns finite values no matter what the sample is!
```python
>>> import pymc4 as pm
>>> dist = pm.BetaBinomial('x', 10, 1., 2.)
2020-02-07 13:29:06.852663: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
>>> dist.log_prob([3., 10.])
<tf.Tensor: shape=(), dtype=float32, numpy=-9.703817>
>>> dist.log_prob([10., 20.])
<tf.Tensor: shape=(), dtype=float32, numpy=-62.71588>
```